ui/components/component-library/modal-body/modal-body.scss

```diff
@@ -0,0 +1,5 @@
+.mm-modal-body {
+  max-height: 100%;
+  overflow-y: auto;
+  position: relative; // Allows for absolute positioning of scroll button
```

Comment:
Eventually I think it would be good if the `ModalBody` handles the display of the scroll button or works well with the `ModalScrollButton` but that would be in a subsequent PR